aten/src/ATen/native/native_functions.yaml

```diff
@@ -776,6 +776,10 @@
   dispatch:
     CPU, CUDA: copysign_
 
+- func: copysign.Scalar_out(Tensor self, Scalar other, *, Tensor(a!) out) -> Tensor(a!)
+  dispatch:
+    CPU, CUDA: copysign_out
```

Comment:
Implementation is 

```
Tensor& copysign_out(const Tensor& self, const Scalar& other, Tensor& result) {
  return at::copysign_out(result, self, wrapped_scalar_tensor(other));
}
```

which is fully dispatched so CPU, CUDA is overly conservative; CompositeImplicitAutograd would be OK. Even better, though, would be to make the kernel structured (not in this PR though https://github.com/pytorch/rfcs/blob/rfc-0005/RFC-0005-structured-kernel-definitions.md )--if you think you are interested in making it structured, no action necessary here, structured will fix this up later.

Reply:
Thank you so much for reviewing this PR, and I'm really interested in the structured kernel.
A new PR #55040 is created for porting `torch.copysign` to structured, please kindly take a look.